### ISSUE TITLE
Automatic width and height or dynamic trigger on Android

### ISSUE THREAD

**Issue by mehmetnyarar** [react-native-picker/picker#170]
I'm trying to render the Picker differently on iOS and Android. For iOS, I show the Picker in a modal and toggle the modal with a Pressable, therefore I don't have any problem there. For Android, I only Render Picker since the dialog/dropdown is built-in. However, without giving a specific width and height,  Picker doesn't show the selected value and it's not possible to change the value. **So, is it possible to determine the width and the height automatically for Android, if they are not specified?**

Similar to iOS, I can wrap the Picker with a Pressable but I don't know how to show/close the dialog/dropdown on Android dynamically. **If I can do that, it can also solve my problem.**

**Comment by iamchrispham** [comment]
I agree there's an issue with the flexibility of the text within the selectable values, and it's subjugated to a row so customization is very limited. A reference to the Picker to expose methods such as open/close the modal/dialog would be key to customization.

**Comment by mateusz1913** [comment]
@Naturalclar after focus events are available, I think it can be closed
